.github/workflows/docker-build.yml

```
@@ -0,0 +1,126 @@
+    - name: Set up JDK 11
+      uses: actions/setup-java@v4
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: maven
+        
+    - name: Build JAR file
+      run: |
+        cd backend
+        mvn clean package -DskipTests
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)